web/src/components/tasks/business-formation/review/section/ReviewSubSection.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactElement, ReactNode } from "react";
+
+interface Props {
+  header: string;
+  marginOverride?: string;
+  children?: ReactNode;
+}
+
+export const ReviewSubSection = (props: Props): ReactElement => {
+  return (
+    <>
+      <div className={`flex space-between ${props.marginOverride ?? ""}`}>
```

Comment:
given that the prop is called `marginOverride` and not `margin`, I think it would make sense to set a "default" margin here instead of empty-string. Still allows people to override margin, which it looks like you do in one or two places? But given that 90% of your usages of this component use "margin-y-4", I would argue for setting that as the default and allowing components that deviate from that norm to override.